### ISSUE TITLE
sql from `copy from sql` can't be successful executed

### ISSUE THREAD

**Issue by Heliner** [dbeaver/dbeaver#15016]
#### System information: 
- Operating system (distribution) and version : Mac os BigSur
- DBeaver version :version 21.3.1.202112130652
- Additional extensions: None

#### Describe the problem you're observing:
copy from sql not get the right sql
#### Steps to reproduce, if exist:
```sql
create table `order`(
	`id` INT UNSIGNED ,
	`date` INT UNSIGNED
);

insert into `order` (id, date) values(1, 20211122),(2, 20211120);

select * from `order`;
```

then 
<img width="651" alt="图片" src="https://user-images.githubusercontent.com/32272517/147355506-fee2e8bd-c1cd-46ab-98d2-b310513e7864.png">

we get an sql : 

```sql
INSERT INTO order (id,`date`) VALUES
	 (1,20211122),
	 (2,20211120);
```

when we execute this sql, we get an error:

```sql
You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order (id,`date`) VALUES
	 (1,20211122),
	 (2,20211120)' at line 1
```

in fact, we expect to get：

```sql
INSERT INTO order (id,`date`) VALUES
	 (1,20211122),
	 (2,20211120);
```

**Comment by ShadelessFox** [comment]
Hello @Heliner,

I don't see any difference between the two code snippets. Is there a mistake?

**Comment by Heliner** [comment]
> #### System information:
> * Operating system (distribution) and version : Mac os BigSur
> * DBeaver version :version 21.3.1.202112130652
> * Additional extensions: None
> 
> #### Describe the problem you're observing:
> copy from sql not get the right sql
> 
> #### Steps to reproduce, if exist:
> ```sql
> create table `order`(
> 	`id` INT UNSIGNED ,
> 	`date` INT UNSIGNED
> );
> 
> insert into `order` (id, date) values(1, 20211122),(2, 20211120);
> 
> select * from `order`;
> ```
> 
> then <img alt="图片" width="651" src="https://user-images.githubusercontent.com/32272517/147355506-fee2e8bd-c1cd-46ab-98d2-b310513e7864.png">
> 
> we get an sql :
> 
> ```sql
> INSERT INTO order (id,`date`) VALUES
> 	 (1,20211122),
> 	 (2,20211120);
> ```
> 
> when we execute this sql, we get an error:
> 
> ```sql
> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order (id,`date`) VALUES
> 	 (1,20211122),
> 	 (2,20211120)' at line 1
> ```
> 
> in fact, we expect to get：
> 
> ```sql
> INSERT INTO order (id,`date`) VALUES
> 	 (1,20211122),
> 	 (2,20211120);
> ```

Sorry, get a mistake:
this ` is conflict with  markdown grammar
in fact, we expect to get：
![image](https://user-images.githubusercontent.com/32272517/147677678-92927e30-eade-418a-a487-f63f6ddc1c02.png)

**Comment by LonwoLonwo** [comment]
Can you please test the Early Access version? https://dbeaver.io/files/ea/

Maybe it was fixed here - #15059

**Comment by uslss** [comment]
thanks for the report.
will be fixed in #15158
